changelog/23.0/23.0.0/summary.md

```diff
@@ -17,6 +19,14 @@
 
 ## <a id="minor-changes"/>Minor Changes</a>
 
+### <a id="deprecations"/>Deprecations</a>
+
+#### <a id="deprecated-metrics"/>Metrics</a>
+
+| Component |        Metric Name        |                     Deprecation PR                      |
+|:---------:|:-------------------------:|:-------------------------------------------------------:|
+| `vtorc`   | `DiscoverInstanceTimings` | [#18406](https://github.com/vitessio/vitess/pull/18406) |
```

Comment:
It is important to state the replacement metric name in the release notes.

Reply:
@timvaillancourt waiting on this to approve & merge your PR

Reply:
@shlomi-noach thanks, updated 👍